projects/application/DecentraBox.md

```diff
@@ -0,0 +1,60 @@
+## Project Name 
+DecentraBox
+
+## Category 
+Application
+
+## Project Description
+With DecentraBox files aren’t stored in centralized data centers— instead, they're encrypted, split into pieces, and distributed on a filecoin+ipfs network.
+
+## Project Status
+under development
+
+## Previews
+Homepage: [https://i.imgur.com/eUN78y8.png](https://i.imgur.com/eUN78y8.png)
+
+Login: [https://i.imgur.com/ykXXIUz.png](https://i.imgur.com/ykXXIUz.png)
+
+Signup: [https://i.imgur.com/aHONeRf.png](https://i.imgur.com/aHONeRf.png)
+
+App: [https://i.imgur.com/ibIPpUr.png](https://i.imgur.com/ibIPpUr.png)  &&  [https://i.imgur.com/XmrVXM2.png](https://i.imgur.com/XmrVXM2.png)
```

Comment:
Could you please update all the pictures since they are not loaded correctly ?

Reply:
Updated the images